website/docs/developer/deployment-guide/testing_deployment.mdx

```diff
@@ -0,0 +1,28 @@
+---
+id: testing_deployment
+title: Testing Deployment
+description: Provides information on testing ORCA code deployment.
+---
+
+import MyImage from '@site/docs/templates/pan-zoom-image.mdx';
```

Comment:
Remove the MyImage import, this is not needed since you don't need the template.